src/main/java/io/github/minecraftcursedlegacy/api/client/TextureRegistry.java

```diff
@@ -0,0 +1,19 @@
+package io.github.minecraftcursedlegacy.api.client;
+
+import io.github.minecraftcursedlegacy.impl.client.TextureRegistryImpl;
+
+public class TextureRegistry {
+	private TextureRegistry() {
+	}
+
+	/**
+	 * Sets a custom texture for an item id.
+	 * Eg:
+	 * io.github.minecraftcursedlegacy.api.client.TextureRegistry.addCustomTexture(280, "assets/modid/tbgn3p08.png"); //280 is the stick id
```

Comment:
Why're you using a fully qualified name here? It makes the javadoc harder to read.